x/concentrated-liquidity/types/migration.go

```diff
@@ -9,6 +9,44 @@ var MigratedIncentiveAccumulatorPoolIDs = map[uint64]struct{}{
 	1431: {},
 }
 
+var MigratedSpreadFactorAccumulatorPoolIDs = map[uint64]struct{}{
```

Comment:
Let's add v25 suffix since we will have more upcoming migrations and have had v23 and v25 inventive migrations.

Will make it easier to track and reason about going forward

Reply:
v25 suffix added https://github.com/osmosis-labs/osmosis/pull/7876/commits/0b77070a7ecafe73f9b0183b8730891395d57c22